src/bindings/python/wheel/setup.py

```diff
@@ -426,6 +459,15 @@ def copy_package_data(self, src_dirs):
             packages.append(WHEEL_LIBS_PACKAGE)
             package_data.update({WHEEL_LIBS_PACKAGE: ["*"]})
 
+    def copy_package_data(self, src_dirs):
+        """Collect package data files from preinstalled dirs and put to the subpackage."""
+        package_dir = os.path.join(PACKAGE_DIR, WHEEL_PACKAGE_DIR)
+        os.makedirs(package_dir, exist_ok=True)
+
+        for src_dir in src_dirs:
+            src, dst = Path(src_dir), Path(package_dir)
+            shutil.copytree(src, dst, dirs_exist_ok=True)
```

Comment:
wonder, whether we can fully rely on `CPACK_GENERATOR=WHEEL` to create a structure of wheel package and avoid any repacking, patchelf calls, etc. E.g. `make install` provides exact content of wheel package with proper layout and RPATH